### PR TITLE
Add job to run testsuite with MSSQL on Windows

### PR DESCRIPTION
**Pull request by ADmad**
Does anybody have ideas how to fix the failing tests for CURL and currency formatting (related to ICU) on Windows?

### CODE REVIEW

**Comment by garas** [comment]
CLDR 36 has new `en_IN` currency format in [CLDR-13097](https://unicode-org.atlassian.net/browse/CLDR-13097). Tests on Linux will start to fail too once ICU/CLDR will be updated.

Curl tests expects not empty content on `http://localhost`, but server responds with empty content. It is unclear what server listens on port 80 on Github Actions Windows VM. Need some more reliable target server, or to skip test.

**Comment by ADmad** [comment]
> CLDR 36 has new en_IN currency format in CLDR-13097. Tests on Linux will start to fail too once ICU/CLDR will be updated.

I updated the tests to not use `en_IN` currency format.

> Curl tests expects not empty content on http://localhost, but server responds with empty content. It is unclear what server listens on port 80 on Github Actions Windows VM. Need some more reliable target server, or to skip test.

I removed the assertion for non empty content. Lack of exception means the request was successful and there's also an assertion for non empty headers which does pass.

Also updated the test for `http://dummy/?sleep` URL to account for both possible error messages on linux and windows.

**Comment by ADmad** [comment]
> Tests: 10129, Assertions: 28788, Skipped: 211, Incomplete: 5

On AppVeyor

> Tests: 10129, Assertions: 29998, Skipped: 179, Incomplete: 5.

On Github, so 32 more tests run :slightly_smiling_face:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,6 +78,38 @@ jobs:
           vendor/bin/phpunit --verbose
         fi
 
+  testsuite-windows:
+    runs-on: windows-2019
+    name: Windows - PHP 7.4 & mssql
```
Comment (othercorey) [inline .github/workflows/ci.yml]:
```suggestion
    name: Windows - PHP 7.4 & SQL Server
```

**Comment by othercorey** [comment]
I can't seem to find what version of SQL Server is provided in the `windows-2019` image. I see the various add-on packages, but not the actual server version.

**Comment by ADmad** [comment]
@othercorey https://github.com/actions/virtual-environments/issues/413#issuecomment-589949674

**Review by othercorey — APPROVED** [review]

**Pull request merged**